clients/v1_shell/table_model.go

```diff
@@ -171,9 +171,12 @@ func (m Model) Init() tea.Cmd {
 	if len(m.tags) > 0 {
 		fmt.Print("Tags: ")
 		for key, val := range m.tags {
-			if key == "" || val == "" {
+			if key == "" {
 				continue
 			}
+			if val == "" {
+				val = " <nil> "
```

Comment:
Why empty spaces around?

Reply:
So that when it is rendered in bubbletea that it will stand out from the equal sign.

Reply:
But, why such a difference? AFAIK, the real values are not rendered with a space in front of it, nor `<nil>` in the table view.

Reply:
It's a question of presentation, which IMHO looks cleaner with a space between the equals sign and the bracket. At this point `val` is only needed for presentation and will not be used further.  However, this is a triviality, so the change has been made.

Reply:
I understand the idea, but we must make compatible and consistent changes.  So, if spaces, the space should be presented on both sides of the equal sign and it should also be added if real value (not placeholder) is presented.